net/urbit/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        urbit urbit 0.4.4 v
+categories          net
+platforms           darwin
+license             MIT
+maintainers         {l2dy @l2dy} openmaintainer
+description         a secure peer-to-peer network of personal servers
+long_description    Urbit is ${description}, built on a clean-slate system software stack.
+homepage            https://www.urbit.org/
+
+checksums           rmd160  babd243f0a2dc1a5900a5914e170f44afccc2a15 \
+                    sha256  3887d8c52b9e576779fd2e9740efc00dbb52316dcd620300eeed1f1c57ef152c
+
+depends_lib         port:gmp \
+                    port:libsigsegv \
+                    path:lib/libssl.dylib:openssl
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:libtool \
+                    port:cmake
+
+patchfiles          patch-Makefile.diff
+
+post-patch {
+    reinplace "s|##PREFIX##|${prefix}|g" ${worksrcpath}/Makefile
+}
+
+use_configure       no
+
+build.args-append   CC=${configure.cc} \
+                    CXX=${configure.cxx} \
+                    CPP=${configure.cpp}
```

Comment:
My comment was not only about +universal. This Portfile will not use any `-arch` flags and therefore will not respect `build_arch` as configured in `macports.conf`. It will always build for the system default.

`${configure.cflags}` would also include`${configure.optflags}` which defaults to `-O2`. Using `-O3` is fine, if upstream prefers to use it.

I do not know anything about `-bind_at_load` or its implications.